app/views/docs/storage.phtml

```diff
@@ -0,0 +1,83 @@
+<p>Appwrite storage allow you to manage files in your project. It provides APIs to upload, download, delete, and list files. Not only that Appwrite storage service provides APIs to manipulate image. Using the prveview endpoint you can crop, resize, rotate, add borders and border radius and select the desired output format for your image.</p>
+
+<p>Appwrite storage service is a simple and easy to use service. You can use it to store images, videos, documents, and other files for your projects.</p>
+
+<p>With version 0.13 of Appwrite, we also have introduced storage buckets as a container for your files so that you can give more context to your files. Storage buckets is similar to Collections we have in our <a href="/database">Database</a> service. But a bucket gives you more control over what types of files you allow, what is the maximum file size you allow, whether or not to encrypt the files.</p>
```

Comment:
```suggestion
<p>With version 0.13 of Appwrite, we also have introduced storage buckets as a container for your files so that you can give more context to your files. Storage buckets is similar to Collections we have in our <a href="/docs/database">Database</a> service. But a bucket gives you more control over what types of files you allow, what is the maximum file size you allow, whether or not to encrypt the files.</p>
```
Broken link